### PR TITLE
[Readme] Update curl info for dev installs

### LINKED ISSUE #6473: [Readme] Update dependency impacting ubuntu dev install - minor tweak
php7.3-curl specifically is required for the 23 release 
- [ ] change `curl` -> `php7.3-curl` in the main LORIS readme

### PR DESCRIPTION
**Pull request by laemtl**
* Since #6187

I have updated the main README.md to add a reference to the php7.3-curl dev dependency.
- [x] Have you updated related documentation?

* Replaces #6474
* Resolves #6473

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Deploy and log in with username *admin* and the password that's set up during de
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.3</b> or higher
+ * curl [production install] or php7.3-curl [development install]
```
Comment (ridz1208) [inline README.md:34]: @johnsaigle do you know if `curl` is necessary ? I can only see the need for php-curl from the linked PR

Comment (johnsaigle) [reply README.md:34]: I think `curl` is required for prod and dev but I can't point to why.

`php-curl` is required only for development installs.

**Comment by johnsaigle** [comment]
I issued a "Request Changes" review on this PR but it's not displaying properly so I'm adding Blocked instead.

**Comment by driusan** [comment]
I agree with @johnsaigle (other than about the his review not showing up, his review shows as changes requested for me..)

**Comment by laemtl** [comment]
Since the 23.0 ubuntu install markdown file has been created in PR#6510, I will apply this change there :)

**Comment by christinerogers** [comment]
closing.  this main readme should perhaps be updated to specify that the Ubuntu actual (dev) readme specs are elsewhere

**Pull request closed without merging**